src/lang/lwc_jaJP.properties

```diff
@@ -9,7 +9,7 @@
 public=共有
 private=個人
 password=パスワード
-kick_trap=KICKトラップ
+kick_trap=キックトラップ
```

Comment:
Can you put miscellaneous translation fixes in a separate PR, as it doesn't directly pertain to this one.